components/apimgt/org.wso2.carbon.apimgt.rest.api.publisher.v1/src/main/java/org/wso2/carbon/apimgt/rest/api/publisher/v1/impl/EndpointCertificatesApiServiceImpl.java

```diff
@@ -226,6 +233,53 @@ public Response updateEndpointCertificateByAlias(String alias, InputStream certi
         return null;
     }
 
+    @Override
+    public Response getCertificateUsageByAlias(String alias, Integer limit, Integer offset, MessageContext messageContext) throws APIManagementException {
+
+        List<API> allMatchedApis = new ArrayList<>();
+        APIQuickInfoListDTO apiQuickInfoListDTO;
+        limit = limit != null ? limit : RestApiConstants.PAGINATION_LIMIT_DEFAULT;
+        offset = offset != null ? offset : RestApiConstants.PAGINATION_OFFSET_DEFAULT;
+        CertificateMetadataDTO certificateMetadataDTO;
+
+        try {
+
+            if (StringUtils.isEmpty(alias)) {
```

Comment:
According to API definition, alias is required path parameter. Hence, it cannot be empty. we can remove this redundant check?

Reply:
Removed